pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -168,7 +168,7 @@ func (m *AddOnManager) String() string {
 }
 
 func (m *AddOnManager) applyAddOn(addOn addon.AddOn, context *command.ExecutionContext) error {
-	fmt.Print(fmt.Sprintf("Applying addon %s:", addOn.MetaData().Name()))
+	fmt.Print(fmt.Sprintf("-- Applying addon '%s':", addOn.MetaData().Name()))
```

Comment:
I think it is important to use the name. The name is the key for the various add-on commands. Also the description line can be long. 

Better to use the echo command to provide more context.